samples/error-correction/syndrome/Syndrome.qs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+namespace Microsoft.Quantum.Samples.ErrorCorrection.Syndrome
+{
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Preparation;
+    open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Convert;
+
+    /// # Summary
+    /// Measure qubit in a given basis and return the result
+    ///
+    /// # Input
+    /// ## basis
+    /// Pauli basis in which to perform measurement
+    /// ## qubit
+    /// Qubit to measure
+    /// 
+    /// # Output
+    /// ## result
+    /// Measurement result
+    operation BasisMeasure(basis: Pauli, qubit: Qubit): Result {
```

Comment:
I'd suggest `MeasureInBasis` to match `PrepareInBasis` below. As a minor nit, we've generally used spaces between identifiers and their types as well:
```suggestion
    operation BasisMeasure(basis : Pauli, qubit : Qubit): Result {
```

Reply:
Thanks, I hadn't realized that I forgot to do this!